test/jellyfish_web/controllers/room_controller_test.exs

```diff
@@ -144,6 +144,25 @@ defmodule JellyfishWeb.RoomControllerTest do
       assert Enum.any?(rooms, &(&1.id == room2_id))
       assert Enum.all?(rooms, &(&1.id != room_id))
     end
+
+    test "roomService removes room on engine crash", %{room_id: room_id} = state do
```

Comment:
I would call this `room closes engine crash` as RoomService doesn't remove room in this test case